packages/slider/src/Slider.ts

```diff
@@ -273,6 +328,22 @@ export class Slider extends ObserveSlotText(SliderHandle, '') {
         this.handleController.beginTrackDrag(event);
     }
 
+    private handleNumberInput(event: Event & { target: NumberField }): void {
+        const { value } = event.target;
+        if (!isNaN(value)) {
+            this.value = value;
+        }
+    }
+
+    private handleNumberChange(event: Event & { target: NumberField }): void {
+        const { value } = event.target;
+        if (isNaN(value)) {
+            this.value = 0;
```

Comment:
Would it be a better idea to fallback on the current value instead of 0?

Reply:
Good question. If we do take this path, it should also take into account `this.min`. However, the idea here is the `<sp-number-field>` returns a `NaN` value when its contents have been deleted and then the empty value committed (either `Enter` key is pressed or the field is blurred). I was thinking of this as an explicit decision by the user to "reset" the value. We don't currently have a _default_ value, so I chose to default it to "nothing" or `this.min`. I can see this going both ways, but feel like the active decision to _delete_ the value is something we should respect. I'll surface this to the steering committed to get some other thoughts.

Reply:
Based off of discussion across the team, particularly with design, to align with other offerings the new experience here will be as follows:
- `input` is no longer tracked
  - only "committed" values will be sent to the slider
  - `input` event from the Number Field will have their propagation stopped
- `change` will be tracked and _if_ the change is to `NaN` the previous "committed" value will be reapplied
- when a _valid_ `change` event is handled, it will dispatch a matching `input` event for consistency